File: C++/프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int price) {
    int answer = 0;
    if (price >= 500000) {
        price = price * 0.8;  
    } else if (price >= 300000) {
        price = price * 0.9;  
    } else if (price >= 100000) {
        price = price * 0.95;
    }

    answer = price;
    return answer;
}

File: JavaScript/프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.js
function solution(price) {
    var answer = 0;
    if(price>=500000){
       price=Math.floor(price*0.8);
    }
    else if(price>=300000){
       price=Math.floor(price*0.9);
    }
    else if(price>=100000){
        price=Math.floor(price*0.95);
    }
    answer=price;
    return answer;
}